lib/src/main/resources/bag/metadata/afm/afm.xsd

```diff
@@ -75,7 +78,11 @@
             <xs:documentation xml:lang="dut">technische bestandsspecificatie</xs:documentation>
         </xs:annotation>
     </xs:element>
-
+    
+    <xs:element name="file_name" substitutionGroup="dcterms:title"/>
+    
+    <xs:element name="description" substitutionGroup="dcterms:description"/>
```

Comment:
Child or sibling of `addmd`?

Reply:
i would say same as the other elements here? maybe i'm not following